CHANGELOG

```diff
@@ -1,3 +1,31 @@
+[v#.#.#] ([month] [YYYY])
+  - Issues:
+    - Display evidence in a table
```

Comment:
Single item lists can be compressed into 1 line for the time being.

Reply:
There are 2 other PR's that will be merging into this one with their own entires into the changelog and this makes the merge smoother.